res/controllers/Numark-Mixtrack-3-scripts.js

```diff
@@ -858,6 +859,7 @@ NumarkMixtrack3.deck = function(decknum) {
     // NMTP3 is a "Model A" controller for scratching, it centers on 0.
     // See http://www.mixxx.org/wiki/doku.php/midi_scripting#scratching
     // and see "Jogger" object constructor
+    this.beatJumpSize = 1;
```

Comment:
In Windows 10 you have to click "More Info" and then tell it you really mean to install it.